### ISSUE TITLE
[internal] Environments run slow on the cluster

### ISSUE THREAD

**Issue by alexander-soare** [huggingface/lerobot#287]
Xarm and Aloha envs run much slower (> 10x) on our cluster nodes.

Here is a benchmarking script:

```python
import time
import gym_aloha
import gym_pusht
import gym_xarm
import gymnasium as gym

env_ids = [
    "gym_pusht/PushT-v0",
    "gym_xarm/XarmLift-v0",
    "gym_aloha/AlohaTransferCube-v0",
]

n_steps = 1000

for env_id in env_ids:
    env = gym.make(
        id=env_id,
        obs_type="pixels_agent_pos",
        render_mode="rgb_array",
        visualization_height=384,
        visualization_width=384,
    )

    env.reset()

    start = time.perf_counter()
    for _ in range(n_steps):
        obs, reward, terminated, truncated, *_ = env.step(env.action_space.sample() / 10)
    print(f"{env_id} time for {n_steps} steps: {time.perf_counter() - start:.3f} sec")

    env.close()
```

Running this on my machine (AMD Ryzen 7 5800X 8-Core Processor, 2 thread per core) I get:

```
gym_pusht/PushT-v0 time for 1000 steps: 1.982 sec
gym_xarm/XarmLift-v0 time for 1000 steps: 0.625 sec
gym_aloha/AlohaTransferCube-v0 time for 1000 steps: 4.995 sec
```

Running on the cluster with `--cpus-per-task=8` I get:
```
gym_pusht/PushT-v0 time for 1000 steps: 2.359 sec
gym_xarm/XarmLift-v0 time for 1000 steps: 33.177 sec
gym_aloha/AlohaTransferCube-v0 time for 1000 steps: 72.524 sec
```

Running on the cluster with `--cpus-per-task=32` I get:

```
gym_pusht/PushT-v0 time for 1000 steps: 2.434 sec
gym_xarm/XarmLift-v0 time for 1000 steps: 38.230 sec
gym_aloha/AlohaTransferCube-v0 time for 1000 steps: 95.093 sec
```

cc @perezjln @aliberts

**Comment by qgallouedec** [comment]
~Good news, it probably comes from the implementation of the environment, look at what you get with lowcostrobot:~

```
ReachCube-v0 time for 1000 steps: 0.094 sec
```
EDIT: my bad, wrong mode

**Comment by alexander-soare** [comment]
News. Try the script with obs_type="state". Xarm goes way faster (0.5 sec) (and Aloha fails due to a NotImplemented error). So it's very likely that the rendering as at fault.

**Comment by qgallouedec** [comment]
Deseperately trying to setup egl to render on the GPU.

**Comment by aliberts** [comment]
Not sure but this seems related: https://github.com/google-deepmind/dm_control/issues/209

TLDR: It's probably coming from the fact that we're using a docker container. Unfortunately, the solution they use - which is to use the [nvidia/cudagl](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/cudagl) images — is obsolete as it is not maintained anymore :/

**Comment by qgallouedec** [comment]
Can we drop docker?
